RELEASING.md

```diff
@@ -2,6 +2,7 @@
 1. Communicate that a release is about to be released in [Gitter Akka Dev Channel](https://gitter.im/akka/dev), so that no new Pull Requests are merged
 2. Add a release notes entry in docs/src/main/paradox/release-notes.md. As a helper run
 `scripts/commits-for-release-notes.sh <last-version-tag>` which will output a list of commits grouped by submodule.
+2. Update latest Akka 2.5 version variable `akka25Version` in `project/Dependencies.scala`
```

Comment:
having 2x 'step 2' looks weird in the markdown (though it's rendered 'correctly')... perhaps just use `1.` for each step?